CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## [v1.15.0] - 2019-05-13
+
+- #231 Add private connection fields to Databases - @mikejholly
```

Comment:
Looks like https://github.com/digitalocean/godo/pull/223 is also going out in this release. Probably ought to mention that ;)

Reply:
Updated